src/main/java/RecrBad.java

```diff
@@ -55,28 +55,53 @@ private static void sayBye() {
         printLine();
     }
 
-    private static String[] addToList(String[] lines, String line) {
+    private static Task[] addToList(Task[] lines, String line) {
         // copies and returns longer String[lines.length+1]
-        String[] moreLines = Arrays.copyOf(lines, lines.length + 1);
-        moreLines[lines.length] = line; //append at last elem
+        Task[] moreLines = Arrays.copyOf(lines, lines.length + 1);
+        moreLines[lines.length] = new Task(line); //append at last elem
 
         printLine();
         System.out.println("You added: " + line);
         return moreLines;
     }
 
-    private static void displayList(String[] lines) {
+    private static void displayList(Task[] lines) {
         int count = 1;
-        for (String line : lines) {
-            System.out.println(count + ". " + line);
+        for (Task line : lines) {
+            System.out.println(count + ".[" + line.getStatus() + "] " + line.description);
             count += 1;
         }
     }
 
+    private static void displayListItem(Task[] lines, int index) {
+            System.out.println("[" + lines[index].getStatus() + "] " + lines[index].description);
+    }
+    private static void markOperation(Task[] lines, String line, boolean isMark){
+        String[] req = line.split(" ");
```

Comment:
Variable name was a little hard to understand at a glance, I'm guessing req is request? Might be better to spell it out in this case